src/components/graph/search/GraphSearchInput.tsx

```diff
@@ -225,7 +225,7 @@ function GraphSearchInput(props: Props) {
 	}
 
 	function handleInputValueChange(event: React.ChangeEvent<HTMLInputElement>) {
-		const value = event.target.value;
+		const value = event.target.value.trim();
```

Comment:
I think there is still feature that allows to enter date, and while entering date e.g `02.11.2021 00:00:00.000` space is required

Reply:
i think trim method is not for getting rid of all spaces but cuts off string's end and start spaces only...
correct me if it is not or clarify what case do you mean

Reply:
try to enter `02.11.2021 00:00:00.000` . you wont be able to insert space after date